src/js/components/Box/StyledBox.js

```diff
@@ -49,6 +49,7 @@ const BASIS_MAP = {
 
 const basisStyle = css`
   flex-basis: ${props => BASIS_MAP[props.basis] || props.theme.global.size[props.basis]};
+  overflow: scroll;
```

Comment:
I don't think this will always be correct. It could be "hidden". Why did we need to add this?

Reply:
this is for the Box going over in the Box doc split